raster-foundry/raster-foundry
# Bump GeoTrellis dependency to 3.4

## Overview

Updates to GeoTrellis 3.4 to fix an extent rounding issue

### Checklist


### Notes

Bonus of this upgrade is now we can get rid of the `spark` dependency entirely because `HttpRangeReader` moved to `geotrellis-utils`!

## Testing Instructions

- Start up server and make sure server works via basic UI interaction (viewing a few projects tiles mostly)

Closes https://github.com/azavea/raster-foundry-platform/issues/1043
